src/utils/withWidth.js

```diff
@@ -13,24 +13,24 @@ import { keys } from '../styles/breakpoints';
  * @param breakpoint
  * @param inclusive - defaults to true
  */
-export const isWidthUp = (screenWidth, breakpoint, inclusive = true) => {
+export const isWidthUp = (breakpoint, screenWidth, inclusive = true) => {
```

Comment:
What's the motivation for changing the signature?
I believe it's used somewhere else. Did you switch the arguments everywhere?